docs/icr_alma.md

```diff
@@ -0,0 +1,41 @@
+# nf-core/configs: Institute of Cancer Research (Alma HPC) Configuration
+
+Deployment and testing of nf-core pipelines on the Alma cluster is on-going in collaboration between the RSE team and research groups.
+
+To run an nf-core pipeline on Alma, run the pipeline with `-profile icr_alma`. This will download and launch the [`icr_alma.config`](../conf/icr_alma.config) which has been pre-configured with a setup suitable for the Alma HPC cluster.
+
+Before running the pipeline you will need to load Nextflow using the environment module system. The preferred way is to create a condo or mamba environment (you only need to do this once) which you activate.
+
+```bash
+## Create and activate mamba environment
+mamba create --name mamba_nf -c bioconda nextflow nf-core
+mamba activate mamba_nf
+```
+
+Singularity is installed on the compute nodes of Alma, but not the login nodes. There is no module for Singularity.
+
+All of the intermediate files required to run the pipeline will be stored in the `work/` directory. It is recommended to delete this directory after the pipeline has finished successfully because it can get quite large.
```

Comment:
If you want this to be cleaned up automatically on a successful run, you can specify `cleanup = true` . If a pipeline fails, the cleanup doesn't occur even if this is set to true to allow you to debug the run